content/cybersecurity/concepts/cyber-attack/terms/cryptojacking/cryptojacking.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Cryptojacking Attack'
+Description: 'Cryptojacking is a type of cyberattack where a hacker uses a target’s computing power to mine cryptocurrency without the target’s knowledge or consent.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+ Once the device is infected, the attacker can use its resources to mine cryptocurrency, which can be very lucrative. The most commonly mined cryptocurrency is Monero, which is preferred by attackers because it is difficult to trace. Cryptojacking can slow down the infected device and cause it to overheat, which can damage the hardware
```

Comment:
```suggestion

**Cryptojacking** is a type of cyberattack where a hacker uses a target’s computing power to mine cryptocurrency without the target’s knowledge or consent. 

Once the device is infected, the attacker can use its resources to mine cryptocurrency, which can be very lucrative. The most commonly mined cryptocurrency is Monero, which is preferred by attackers because it is difficult to trace. Cryptojacking can slow down the infected device and cause it to overheat, which can damage the hardware.
```

In addition, before "Once the device is infected," we seem to be missing a sentence or two about how the device is infected.